scripts/shared.sh

```diff
@@ -6,14 +6,21 @@ set -eu -o pipefail
 
 function _extract_cluster_config_value() {
     # Try extracting config from cluster config.json, default config.json, then aladdin config.json
-    local value
-    value="$1"
+    local path default value
+    path="$1"
+    default="${2:-}"
     if [[ -z "${ALADDIN_CONFIG_DIR:-}" ]]; then
         return 0
     fi
-    jq -nr --arg value "$value" 'first(inputs | (if .[$value] == null then empty else .[$value] end))' \
+    value=$(jq -nr "first(inputs | (if .$path == null then empty else .$path end))" \
```

Comment:
changed from bracket notation to string interpolation to allow nested path access